2_Controllers/test_controller.js

```diff
@@ -1,12 +1,19 @@
-const { fetchTopics, fetchEndpoints } = require("../1_Models/test_model");
-const { request, response } = require("../app");
+const {
+  fetchTopics,
+  fetchEndpoints,
+  fetchArticlesById,
+} = require("../1_Models/test_model");
+// const { request, response } = require("../app");
+const articles = require("../db/data/test-data/articles");
 const topics = require("../db/data/test-data/topics");
 
 const getTopics = (request, response, next) => {
-  fetchTopics().then((topics) => {
-    // console.log("model is in controller");
-    response.status(200).json({ topics });
-  });
+  fetchTopics()
+    .then((topics) => {
+      // console.log("model is in controller");
```

Comment:
remove console.logs as they clutter up our code, and we should make sure we remove them as soon as we can.